src/patternfly/components/AppLauncher/examples/app-launcher-expanded-example.hbs

```diff
@@ -1 +1,8 @@
-{{#> app-launcher id="app-launcher-example-expanded" app-launcher--IsExpanded="true"}}{{/app-launcher}}
\ No newline at end of file
+{{#> app-launcher id="app-launcher-example-expanded" app-launcher--IsExpanded="true"}}
+  {{#> app-launcher-menu}}
+    <li>{{#> app-launcher-menu-item app-launcher-menu-item--attribute='href="#"'}}Link one{{/app-launcher-menu-item}}</li>
```

Comment:
Instead of "Link one, Action two, Div three", I think you could just list "Link, Action, Div". Or if you want to include the numbers, it would be like "Link 1, Link 2, Action 1, Action 2" etc.